ui/src/app/cohort-search/constant.ts

```diff
@@ -110,7 +110,7 @@ export const MODIFIERS_MAP = {
     },
   },
   [ModifierType.NUMOFOCCURRENCES]: {
-    name: 'Has Occurrences',
+    name: 'Number of occurrence dates',
```

Comment:
We should capitalize everything here and below to match the other modifiers
```suggestion
    name: 'Number Of Occurrence Dates',
```